webapp/channels/src/components/admin_console/admin_sidebar/admin_sidebar_category.tsx

```diff
@@ -20,7 +20,7 @@ const AdminSidebarCategory = ({icon, title, action, children, definitionKey, nam
     let link = parentLink;
     let titleDiv = (
         <div className='category-title category-title--active'>
-            {icon}
+            <span className='category-icon'>{icon}</span>
```

Comment:
What's this change for?

Reply:
This is because we need to remove the `className` parameter of the Icons in the admin console to pass the type checking, and this was the solution for that.